docs/source/tensorbay_cli/getting_started_with_cli.rst

```diff
@@ -22,25 +22,43 @@ To use TensorBay CLI, please install TensorBay SDK first.
    $ pip3 install tensorbay
 
 ***************
- Configuration
+ Authentication
 ***************
 
 An accessKey_ is used for identification when using TensorBay to operate datasets.
 
 .. _accesskey: https://gas.graviti.cn/tensorbay/developer
 
-Set the accessKey into configuration:
+Set the accessKey into the default profile:
 
 .. code:: html
 
    $ gas auth [ACCESSKEY]
 
-To show authentication information:
+Set the accessKey into the specific profile:
```

Comment:
Add `profiles` section at the end of the file, and move the newly added to the `profiles` section.